### PR TITLE
Add ga in admin to improve Strapi

### PR DESCRIPTION
**Pull request by lauriejim**

### CODE REVIEW

File: packages/strapi-admin/admin/src/containers/AdminPage/index.js [hunk 1]
```diff
@@ -71,11 +72,15 @@ export class AdminPage extends React.Component { // eslint-disable-line react/pr
...
     this.checkLogin(this.props);
+
+    ReactGA.initialize('UA-54313258-9');
   }
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.location.pathname !== this.props.location.pathname) {
       this.checkLogin(nextProps);
+
+      ReactGA.pageview(window.location.pathname);
```
Comment (soupette) [inline packages/strapi-admin/admin/src/containers/AdminPage/index.js]: `ReactGA.pageview(nextProps.location.pathname)`

File: packages/strapi-admin/admin/src/containers/AdminPage/index.js [hunk 2]
```diff
@@ -71,11 +72,15 @@ export class AdminPage extends React.Component { // eslint-disable-line react/pr
 
   componentDidMount() {
     this.checkLogin(this.props);
+
+    ReactGA.initialize('UA-54313258-9');
```
Comment (abdonrd) [inline packages/strapi-admin/admin/src/containers/AdminPage/index.js]: Will it be configurable? Or you will track all the Strapi projects?

Comment (ArkhipovK) [reply packages/strapi-admin/admin/src/containers/AdminPage/index.js]: Sure thing. What if i dont want to track my project?

Comment (lauriejim) [reply packages/strapi-admin/admin/src/containers/AdminPage/index.js]: Yap we plan to add a configuration to disable it of course ;)

Comment (lauriejim) [reply packages/strapi-admin/admin/src/containers/AdminPage/index.js]: Done here https://github.com/strapi/strapi/pull/834

**Comment by lauriejim** [comment]
@asharamseervi this is an open source project, we *offer* you a software with the source code. If there is something you don't want, feel free to remove lines of code of your project.

We can speak a long time about what is fair and what is not fair with open source projects. 
Using a software freely and not helping to make it better or try to understand how it is used in order to to be able to define a better roadmap and to make the project better ?

In this case, you are completely able (as asked @abdonrd) to add a configuration to disable this and submit a PR: what we already did it here https://github.com/strapi/strapi/pull/834.

**Pull request merged**